src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -426,7 +428,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseHttpsRedirection();
 
+            // Add default wwwroot static files bindings
             app.UseStaticFiles();
+
+            // Add additional bindings
+            var fileExtensionsBindings = Configuration.GetSection("Content").GetSection("FileExtensionsBindings").Get<Dictionary<string, string>>();
```

Comment:
please consider of using such extension https://github.com/VirtoCommerce/vc-storefront/blob/master/VirtoCommerce.Storefront/Controllers/AssetController.cs#L92 rather than  increasing the complexity of  app settings